controllers/commentController.js

```diff
@@ -148,7 +140,161 @@ export default class CommentController {
       }
       res.status(OK).json({ comment });
     } catch (error) {
-      res.status(INTERNAL_SERVER_ERROR).json({ message: "Sorry, this is not working properly. We now know about this mistake and are working to fix it" });
+      res.status(INTERNAL_SERVER_ERROR).json({
+        message:
+          "Sorry, this is not working properly. We now know about this mistake and are working to fix it"
+      });
+    }
+  }
+
+  /**
+   * @description Allows users to fetch a particular comment of an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+  static async fetch(req, res) {
+    try {
+      const { id: commentId } = req.params;
+      const comment = await Comment.findOne({ where: { id: commentId } });
+
+      if (!comment) {
+        return res
+          .status(BAD_REQUEST)
+          .json({ message: "The comment does not exist" });
+      }
+      res.status(OK).json({ comment });
+    } catch (error) {
+      res.status(INTERNAL_SERVER_ERROR).json({ message: error.message });
+    }
+  }
+
+  /**
+   * @description Allows users to fetch all comments of an article.
+   * @param  {Object} req - The request object.
+   * @param  {Object} res - The response object.
+   * @returns {Object} - It returns the request response object.
+   */
+  static async fetchAllComments(req, res) {
+    try {
+      const { slug } = req.params;
+
+      const article = await Article.findOne({
+        where: { slug },
+        include: [
+          {
+            model: Comment,
+            attributes: ["body"],
```

Comment:
- <code>CommentId</code> is highly needed so that user can be able to grab it and access comment history. 
- Also attributes such as createdAt, and updated are needed since we may want to display them on the UI.

Reply:
- Don't you think comment likes are also needed?

Reply:
@YvesIraguha It might happen to have an article with thousands or even millions of comments, we need to paginate comments in order to give the user a better experience by optimizing the size of the response body to always be less than 200KB and that also reduces the server overload. 
Imagine a user having to wait for more than a minute just to see comments because we are returning all of them at once.